app/models/invoice.rb

```diff
@@ -120,6 +120,14 @@ def subtotal_before_prepaid_credits
     amount + wallet_transaction_amount
   end
 
+  def issued_at
+    issuing_time = created_at.in_time_zone(customer.applicable_timezone)
```

Comment:
We should not expose values in customer timezone, all datetime should be exposed as UTC. It's the front-end responsibility to do the conversion into the right timezone.

Invoices have a `timezone` attribute which can be exposed in the GraphQL type to help the front-end with this conversion.